src/core/maprenderer/qgsmaprendererjob.cpp

```diff
@@ -106,6 +106,7 @@ LayerRenderJob::LayerRenderJob( LayerRenderJob &&other )
   , estimatedRenderingTime( other.estimatedRenderingTime )
   , errors( other.errors )
   , layerId( other.layerId )
+  , maskPainter( nullptr ) // should this be other.maskPainter??
```

Comment:
Assuming you're not seeking an answer now? :)

Reply:
I've left it at the current behavior, since things seem to be working ok with the null value. It's more a debugging hint to someone in future to draw attention if there is some related issue around here...

Reply:
Good move.